project/Dependencies.scala

```diff
@@ -17,9 +17,8 @@ object Dependencies {
 
   val InfluxDBJavaVersion = "2.15"
 
-  val AwsSdkVersion = "1.11.476"
-  val AwsSdk2Version = "2.10.60"
-  val AwsSpiAkkaHttpVersion = "0.0.7"
+  val AwsSdk2Version = "2.11.3"
+  val AwsSpiAkkaHttpVersion = "0.0.8"
```

Comment:
The Netty upgrade motivated the 2.10 to 2.11 update. https://github.com/aws/aws-sdk-java-v2/pull/1692
By default Alpakka excludes the Netty client from the AWS connectors and uses Akka HTTP via https://github.com/matsluni/aws-spi-akka-http/